### PR TITLE
Update user_preferences_extra_conf.yml first try cmems credentials ad…

### PR DESCRIPTION
**Pull request by yvanlebras**
…dition

### CODE REVIEW

File: files/galaxy/config/user_preferences_extra_conf.yml [hunk 1]
```diff
@@ -223,6 +223,14 @@ preferences:
         type: password
         required: False
 
+  cmems_account:
+    description: Your Copernicus CMEMS API Key (Copernicus Marine Service API Key)
+    inputs:
+      - name: cmems_username
```
Comment (bgruening) [inline files/galaxy/config/user_preferences_extra_conf.yml:229]: You need a name and a password isn't it?

Comment (yvanlebras) [reply files/galaxy/config/user_preferences_extra_conf.yml:229]: True. I made an update. Not sure it is cmems_username and cmems_password or CMEMS_USERNAME and CMEMS_PASSWORD (in upper case)

Comment (yvanlebras) [reply files/galaxy/config/user_preferences_extra_conf.yml:229]: Upper case is used for os.environ on the python script

Comment (yvanlebras) [reply files/galaxy/config/user_preferences_extra_conf.yml:229]: https://github.com/galaxyecology/tools-ecology/pull/83/files#diff-f04a544c4e091c01dbb1204d6542b6aa5c143fdbb545283a6471de3f83db40d4R59

Comment (bgruening) [reply files/galaxy/config/user_preferences_extra_conf.yml:229]: But this is what Galaxy uses: https://github.com/galaxyecology/tools-ecology/pull/83/files#diff-135d08b2c88f38c565c73c2f2ee959d66a4520c676eaa880c6b0f8d9a336d7e1R33

Its a bit unfortunate that those are different.

**Review by bgruening — APPROVED** [review]

**Pull request merged**